AbdulKhaliq59/capstone-backend
# Feat modify usermodel controller

#### What does this PR do?
This PR is used to modify something which was not good
#### Description of Task to be completed?
- modify `User` schema
- Modify `Create User` controller
- modify `signin` controller
- create `account verification` controller
#### How should this be manually tested?
 - `clone` this repository
 - run `npm run start`
 - Check on `/signup` route
 - Check on `/verify_account` route
 - Check on `/signin` route
#### Any background context you want to provide?
- `clone` this repository
#### What are the relevant pivotal tracker stories?
#### Screenshots (if appropriate)
#### Questions:
